### ISSUE TITLE
Profile Removal or DMCA Takedown request.

### ISSUE THREAD

**Issue by PeterEnglert** [spikecodes/libreddit#358]
## How does this feature work on Reddit?

DMCA Takedown request applies as a basic functionality.
## Describe the implementation into Libreddit

 DMCA Takedown form for Libreddit and/or a simple Profile removal request form would be appreciated. 
## Describe alternatives you've considered

  Either that or a support email you can contact for individual requests for those who do not want their posts mirrored by another site.
## Additional context

  I would like my profile removed from libreddit. There is no email, DMCA request, or anything of the sort available. I am sure others would appreciate the transparency as well the ability to remove DMCA media/their profile on request.
#

**Comment by clydebottoms** [comment]
libreddit doesn't have profiles though?

**Comment by Adithr123** [comment]
It should. That would be interesting

**Comment by clydebottoms** [comment]
there's like 8 settings what use would a profile provide other than causing issues like this?

**Comment by r7l** [comment]
libreddit doesn't even act as mirror like archive.org. From what i've seen (running a private instance) it doesn't even store anything. If something is gone from Reddit, it is also gone from libreddit as well. So it's basically just a fancy tunnel to Reddit. If anyone is bothered by any content visible, Reddit would be the place to send complains to.

**Comment by PeterEnglert** [comment]
First off I'd like to thank everyone's input here. 
I have browsed with libreddit and viewed my own profile mirrored from reddit on this site; for some reason it still retains media/comment posts that have been removed or deleted by the original owner on the main site. If this is something that should be consulted on reddit's end, why is it showing up exclusively on libreddit even when the original reddit post has been deleted?
